exporter/prometheusexporter/go.mod

```diff
@@ -143,6 +143,8 @@ replace github.com/open-telemetry/opentelemetry-collector-contrib/internal/corei
 
 replace github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/opencensus => ../../pkg/translator/opencensus
 
+replace github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/prometheusremotewrite => ../../pkg/translator/prometheusremotewrite
```

Comment:
Is this needed?  It doesn't look like a requirement for this module was added.

Reply:
Its required for the end to end test:

```
╭─ ~/g/s/g/o/opentelemetry-collector-contrib/exporter/prometheusexporter export-rw-converter                                                                                      ✘ INT 4s
╰─❯ go mod why github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/prometheusremotewrite
# github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/prometheusremotewrite
github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusexporter
github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusexporter.test
github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver
github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver/internal
github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver/internal.test
github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusremotewriteexporter
github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/prometheusremotewrite
```

Reply:
There should then be a `require ... // indirect` directive for it as well, I'd expect.

Reply:
Should I add it? 

The only reason I had to add the `replace` is because `go mod tidy` was failing to find a version for the import. I added this and ran a `go mod tidy` and it didn't add a require directive.